platform/android/java/app/config.gradle

```diff
@@ -46,6 +46,40 @@ ext.getExportVersionName = { ->
 
 final String PLUGIN_VALUE_SEPARATOR_REGEX = "\\|"
 
+// get the list of ABIs the project should be exported to
+ext.getExportEnabledABIs = { ->
+    String enabledABIs = "armeabi-v7a|arm64-v8a|x86|x86_64|";
+    if (project.hasProperty("export_enabled_abis")) {
+        enabledABIs = project.property("export_enabled_abis");
+        if (enabledABIs == null || enabledABIs.isEmpty()) {
+            enabledABIs = "armeabi-v7a|arm64-v8a|x86|x86_64|";
+        }
+    }
```

Comment:
You should use this pattern instead:
```
    String enabledABIs = project.hasProperty("export_enabled_abis") ? project.property("export_enabled_abis") : "";
    if (enabledABIs == null || enabledABIs.isEmpty()) {
       enabledABIs = "armeabi-v7a|arm64-v8a|x86|x86_64";
    }
```

This way you avoid repeating the assignment for the default value. It's preferable because it avoid subtle errors where someone may change the first assignment to the default value, but forget to update the second assignment.
With the suggested pattern, only one location need to be updated.